wiki/Sitemap/en.md

```diff
@@ -110,6 +110,7 @@ tags:
 -   [Internet Relay Chat](/wiki/Internet_Relay_Chat)
 -   [Legal](/wiki/Legal)
     -   [Copyright](/wiki/Legal/Copyright)
+        -   [Music Licensing](/wiki/Legal/Copyright/Music_Licensing)
```

Comment:
```suggestion
    -   [Music Licensing](/wiki/Legal/Copyright/Music_Licensing)
```

shouldn't be further indented

Reply:
oh i guess that's how the pages have been structured in directory structure too.. seems wrong.

Reply:
it's a page under Copyright, this is how the rest of the sitemap is already formatted (at this point https://github.com/ppy/osu-web/pull/5152 would be the place to suggest changes to its style)

Reply:
yeah looks like it could be moved up to Legal. Music Licensing doesn't assume you've read Copyright